packages/create-schemas/tests/e2e.test.ts

```diff
@@ -27,6 +30,47 @@ describe.concurrent("e2e", () => {
         timeout
     );
 
+    test(
+        "officevice.yaml / file URLs",
+        async ({ expect, onTestFinished }) => {
+            const tempFolder = await createTemporaryFolder({
+                onTestFinished,
+            });
+
+            const source = pathToFileURL(join(dataFolder, "officevice.yaml"));
+            const output = pathToFileURL(join(tempFolder, "output.ts"));
+
+            await runCompiledBin({
+                source: source.toString(),
+                output: output.toString(),
+            });
+
+            const result = await readFile(output, "utf-8");
+
+            expect(result).toMatchSnapshot();
+        },
```

Comment:
minor, but any gain in reusing some code between tests here?

Reply:
I second this

Reply:
I'll try to cook something ⌛

Reply:
Made the test declarations more compact:
- Removed the `source` and `output` variables. We can declare the values directly as a parameter to `runCompiledBin`.
- `runCompiledBin` now returns the result. One less problem to worry about.